backend/src/openarchiefbeheer/destruction/tests/endpoints/test_destruction_list_items.py

```diff
@@ -0,0 +1,223 @@
+            destruction_list=destruction_list,
+            processing_status=InternalStatus.processing,
+            with_zaak=True,
+        )
+        item3 = DestructionListItemFactory.create(
+            status=ListItemStatus.suggested,
+            destruction_list=destruction_list,
+            processing_status=InternalStatus.failed,
+            with_zaak=True,
+        )
+
+        self.client.force_authenticate(user=record_manager)
+        endpoint = furl(reverse("api:destruction-list-items-list"))
+        endpoint.args["item-destruction_list"] = str(destruction_list.uuid)
+
+        response = self.client.get(
+            endpoint.url,
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        data = response.json()
+
+        self.assertEqual(data["results"][0]["pk"], item3.pk)
+        self.assertEqual(data["results"][1]["pk"], item2.pk)
+        self.assertEqual(data["results"][2]["pk"], item1.pk)
+
+    @tag("gh-471")
+    def test_item_with_extra_zaak_data(self):
+        record_manager = UserFactory.create(username="record_manager")
+
+        destruction_list = DestructionListFactory.create()
+        DestructionListItemFactory.create(
+            with_zaak=True,
+            zaak__url="http://localhost:8003/zaken/api/v1/zaken/eafc5f37-4524-43ce-872f-39ff3df11e1e",
+            destruction_list=destruction_list,
+            status=ListItemStatus.suggested,
+            processing_status=InternalStatus.failed,
+            extra_zaak_data={
+                "url": "http://localhost:8003/zaken/api/v1/zaken/eafc5f37-4524-43ce-872f-39ff3df11e1e",
+                "zaaktype": {
+                    "url": "http://localhost:8003/catalogi/api/v1/zaaktypen/be210495-20b6-48ff-8d3d-3e44f74c43a4",
+                    "omschrijving": "brand world-class initiatives",
+                    "selectielijst_procestype": {"nummer": 1},
+                },
+                "einddatum": "2024-08-28",
+                "resultaat": None,
+                "startdatum": "2024-07-14",
+                "omschrijving": "",
+                "identificatie": "ZAAK-ID-89",
+            },
+            internal_results={
+                "traceback": "",
+                "created_resources": {},
+                "deleted_resources": {},
+                "resources_to_delete": {},
+            },
+        )
+
+        self.client.force_authenticate(user=record_manager)
+        endpoint = furl(reverse("api:destruction-list-items-list"))
+        endpoint.args["item-destruction_list"] = str(destruction_list.uuid)
+
+        response = self.client.get(
+            endpoint.url,
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        data = response.json()
+
+        self.assertEqual(
+            data["results"][0]["extraZaakData"]["url"],
+            "http://localhost:8003/zaken/api/v1/zaken/eafc5f37-4524-43ce-872f-39ff3df11e1e",
+        )
```

Comment:
This is the new test, the others were just moved from test_endpoints.py

Reply:
I'm not sure what is tested here, the issue describes removing something while this PR seems to take extra data into account?